src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -293,6 +296,26 @@ fun processMacroSimpleResolveVariants(element: RsMacroBodySimpleMatching, proces
     return simple.any { processor(it) }
 }
 
+fun processMetaItemResolveVariants(element: RsMetaItem, processor: RsResolveProcessor): Boolean {
+    if (element.parentOfType<RsMetaItem>()?.identifier?.text != "derive") return false
+    val traitName = element.identifier.text
+    val stdTrait = STD_DERIVE_TRAITS[traitName]
```

Comment:
Or I can pass all not std traits with `traitName` as resolving result and if there are more than one variant user will be able to choose appropriate declaration and in future, we'll probably implement more precise resolving. I think it's better than without resolving at all.